test/unified_format.py

```diff
@@ -1804,14 +1818,28 @@ def check_events(self, spec):
             else:
                 assert server_connection_id is None
 
+    def process_ignore_messages(self, ignore_logs, actual_logs):
+        final_logs = []
+        for log in actual_logs:
+            ignored = False
+            for ignore_log in ignore_logs:
+                if log["data"]["message"] == ignore_log["data"][
+                    "message"
+                ] and self.match_evaluator.match_result(ignore_log, log, test=False):
+                    ignored = True
+                    break
+            if not ignored:
+                final_logs.append(log)
+        return final_logs
+
     def check_log_messages(self, operations, spec):
         def format_logs(log_list):
             client_to_log = defaultdict(list)
             for log in log_list:
                 if log.module == "ocsp_support":
                     continue
                 data = json_util.loads(log.message)
-                client = data.pop("clientId")
+                client = data.pop("clientId") if "clientId" in data else data.pop("topologyId")
```

Comment:
```suggestion
                client = data.pop("clientId", data.pop("topologyId"))
```

Reply:
This is slick, nice.

Reply:
Sadly doesn't work though, it tries to pop the default every time and causes a `KeyError`.